lib/manageiq/appliance_console/cli.rb

```diff
@@ -176,6 +178,10 @@ def run
       install_certs if certs?
       extauth_opts if extauth_opts?
       set_server_state if set_server_state?
+    rescue CliError => e
+      say(e)
```

Comment:
I was expecting this to be `say e.message`

Do they have the same output?

Reply:
@kbrock Thank you. Yes both `say(e) `and `say(e.message) `produce the same output. I'll update it to `say(e.message)` as I agree it is clearer.